tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -300,6 +300,22 @@ var CESIUM_MAP_VIEW = (function() {
         m_viewer.trackedEntity = entity;
     }
 
+    // Set Cesium primitives
+    cesium_load_primitives = function()
+    {
+        var $map_element = $('#' + m_map_target);
+        var m_primitives_options = [];
```

Comment:
Define `m_primitive_options` on line ~25, so that it is globally available to the entire enclosure.